src/roadmap/models.py

```diff
@@ -25,7 +25,7 @@ class LifecycleType(StrEnum):
 
 class SupportStatus(StrEnum):
     supported = "Supported"
-    near_retirement = "Support ends within 3 months"
+    near_retirement = "Support ends within {number} months"
```

Comment:
This is a bit of a hack, using a format string as an enum.

The problem currently is the number is in the enum value. I would like to remove the number from the enum value and make it simply "Near retirement". This would require UI code changes and it also obscures the value considered when calculating the near retirement status.

If we are ok with API just returning "near retirement" and documenting how that is calculated, I would be happy to change the code.

Reply:
Had some discussion and the UI folks are ok with returning "Near retirement" instead of including the month value in the string. I'll work on that change.